### ISSUE TITLE
[Android] Can't build with RN 0.57.0

### ISSUE THREAD

**Issue by gaykov** [react-native-kit/react-native-track-player#281]
After adding `react-native-track-player` to freshly created react native project with the latest RN version (0.57.0) the app is not building and the build gives the following errors:

![image](https://user-images.githubusercontent.com/9068073/45546087-e4b1ef80-b824-11e8-87ec-9be9b670494b.png)

## Steps to reproduce:
- `react-native init TestPlayer`
- `cd TestPlayer`
- `yarn add react-native-track-player`
- `react-native link`
- `react-native run-android` or open the project via Android Studio

## Versions
- "react-native": "0.57.0"
- "react-native-track-player": "^0.2.5"

**Comment by oleksii-sauliak** [comment]
Same issue. I found some workaround. I just added `compile 'com.android.support:support-v4:27.0.0'` to build.gradle file `node_modules/react-native-track-player/android/build.gradle`
```
dependencies {
    ...
    compile 'com.android.support:support-v4:27.0.0'
}
```
I know, it's not best practice to update code in node_modules, but my app works fine now.

**Comment by dcvz** [comment]
This was solved as part of release v1.0.0. Please use that version instead! 👍
